src/guide/advanced/intro.md

```diff
@@ -121,12 +118,9 @@ use iroha_wasm::data_model::prelude::*;
 
 #[iroha_wasm::iroha_wasm]
```

Comment:
I guess we can rename to `entrypoint` for next release?

Reply:
Sounds good. Add it as a good first issue, since we'll have a newbie on the team soon.